bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "echo-angular",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
You don't have to bump this up a major version unless consumers of this service have to make changes.